src/components/HomeSection.jsx

```diff
@@ -5,61 +5,62 @@ import { Container, Grid, Typography } from '@mui/material';
 const HomeSection = ({
```

Comment:
was thinking of renaming this component to something more generic since we can use this in other pages. 
something like `contentGridSection` or `gridSection`

Reply:
What other pages were you thinking? AFAIK we only use this on the home page and we may as well make this an unexported component in `PublicHome` 😛

Reply:
We could use it for the top section in the `About page` but that piece of code is not that complex, so it might not be necessary.